frontend/src/components/forms/GeneralInfo.js

```diff
@@ -1,6 +1,10 @@
 import React from 'react';
```

Comment:
It would be better if you import useState to avoid writing Rect. many times. I think your code will look cleaner
import React, { useState } from 'react';

Reply:
solved on latest commit